src/main/java/com/eternalcode/discordapp/review/command/child/ListChild.java

```diff
@@ -0,0 +1,38 @@
+package com.eternalcode.discordapp.review.command.child;
+
+import com.eternalcode.discordapp.review.GitHubReviewService;
+import com.jagrosh.jdautilities.command.SlashCommand;
+import com.jagrosh.jdautilities.command.SlashCommandEvent;
+import net.dv8tion.jda.api.EmbedBuilder;
+
+import java.util.List;
+import java.util.Map;
+
+public class ListChild extends SlashCommand {
+
+    private final GitHubReviewService gitHubReviewService;
+
+    public ListChild(GitHubReviewService gitHubReviewService) {
+        this.name = "list";
+        this.help = "List all users in review system";
+
+        this.gitHubReviewService = gitHubReviewService;
+    }
+
+    @Override
+    public void execute(SlashCommandEvent event) {
+        try {
+            List<Map.Entry<String, Long>> listOfUsers = this.gitHubReviewService.getListOfUsers();
```

Comment:
Zdecydowanie brakuje jakiegoś obiektu DiscordUser, aby nie używać entry

Reply:
Tylko ze to nie jest powiązane tylko z discordem, mapka przyjmuje id diacordowe i nazwę na gh. 

Tbh co złego w użyciu entry?

Reply:
Zrobie obiekt ReviewUser i go zaimplementuje 😄